geem/static/geem/js/geem_form.js

```diff
@@ -1210,7 +1210,6 @@ function check_entity_id_change(resource_callback = null, entity_callback = null
 		derender_entity_form();
 		reset_specification_tab();
 		top.focusEntityId = null;
-		$('a[href$="panelContent"]').click();
```

Comment:
This line prevents future scenarios where we may want to reload the portal with a different panel selected. I experienced this as I was playing around with ways to reload the validation panel.

I have added an "active" attribute to ``#panelContent`` in ``portal.html``, which accomplishes the same thing as this line, but due to Django's templating abilities, this solution can be easily generalized in the future to allow us to pick and choose the panel of focus when reloading portal.

Reply:
does the href$ need a # in "#panelContent" to match? Otherwise its expecting a tag named "panelContent"?

Reply:
It was working fine.

Reminder--this line was taken out, not added.

Reply:
oh, duh!  Thx.